test_send_flow.js

```
@@ -0,0 +1,311 @@
+
+const http = require('http');
+
+const NODE_URL = 'http://localhost:9850';
+const SENDER = '0x05744314501c42a31c6a5781d5886ef9';
+const RECEIVER = '0x3241aa645dbc949dea59d14baca26461';
+const PUBLIC_KEY = '0xa701e45abf99a29b47f31f46e7000d2036415fbd09ecb04a43b4c99a29e3c3c2';
+
+function rpcCall(method, params = []) {
+  return new Promise((resolve, reject) => {
+    const body = JSON.stringify({ jsonrpc: '2.0', method, params, id: 1 });
+    const req = http.request(NODE_URL, {
```

ESLint

[error] Replace `NODE_URL,` with `⏎······NODE_URL,⏎·····`

(prettier/prettier)